Gemfile

```diff
@@ -38,3 +38,14 @@ else
   gem "bundler", ">= 2.0"
   gem "sqlite3", "~> 1.4.1"
 end
+
+sorbet_version = ENV["SORBET_VERSION"]
```

Comment:
I don't think `SORBET_VERSION` should be defined to a specific version in Travis. That will result in way too much churn. Instead I'd suggest that we have two versions of this: Sorbet from Rubygems (so, latest release) and sorbet/sorbet's master branch directly from GitHub.

However, at least for now, these are exactly the same: Each commit to sorbet/sorbet appears to be deployed directly to rubygems. (The current version is 0.4.4389, which lines up with 4,389 commits to master.) So for now, we probably just need to test against Sorbet as coming from Rubygems, as we were doing already, but we should probably expect their release process to change in the future.

Reply:
I guess my point is, for as long as the latest version of the Sorbet gem is going to be their HEAD commit, we probably should just always test on that version, rather than also testing on a known earlier version.